slackblocks/elements.py

```diff
@@ -433,9 +433,9 @@ class Image(Element):
 
     def __init__(
         self,
-        alt_text: str,
         image_url: Optional[str] = None,
         slack_file: Optional[SlackFile] = None,
+        alt_text: str = " ",
```

Comment:
@nicklambourne Do you want this type to also be `Optional`?

Reply:
No, that should be fine as is. I'm a little worried about changing the order of the arguments, as that technically makes it a breaking change and anyone who wasn't using kwargs will now have a broken build.